src/components/super-spawner.js

```diff
@@ -0,0 +1,40 @@
+AFRAME.registerComponent("super-spawner", {
+  schema: {
+    template: { default: "" },
+    spawn_position: { type: "vec3" }
```

Comment:
Multi-word AFrame component properties should use camel case instead of snake case.